src/app/components/layout/navbar/navbar.component.ts

```diff
@@ -155,34 +156,31 @@ export class NavbarComponent implements OnInit, AfterViewInit, OnDestroy {
           northboundServices = sortBy(northboundServices, function (obj) {
             return obj.name.toLowerCase();
           });
-          const notificationService = servicesData.filter((el => (el.type === 'Notification')));
-          const managementService = servicesData.filter((el => (el.type === 'Management')));
-          const dispatcherService = servicesData.filter((el => (el.type === 'Dispatcher')));
-          const bucketStorageService = servicesData.filter((el => (el.type === 'BucketStorage')));
+
+          const currentService = JSON.parse(sessionStorage.getItem('SERVICE_NAME'));
+          let expectedExternalServiceType = ['Notification', 'Dispatcher'];
+          if (currentService === 'FogLAMP') {
+            expectedExternalServiceType = ['Notification', 'Management', 'Dispatcher', 'BucketStorage'];
+          }
+          const additionalServices = servicesData.filter((s) => expectedExternalServiceType.includes(s.type))
+
+          // If service is failed or unresponsive, check schedule as well
+          const failedOrUnresponsiveServices = additionalServices.filter((s) => ['failed', 'unresponsive'].includes(s.status.toLowerCase()));
```

Comment:
What if a service was shutdown and Fledge restarted?

Reply:
On navbar hover list, we are showing services which are coming in /service API response. So, if it is not in the response (in the above^ case), it will not be visible there.

Do you mean in that case when service is not available in /service API, we need to make call for its schedule to show in the hover list?

Reply:
Nothing is changed beyond the JIRA scope here.

If any service is failed or unresponsive, we need to check schedule status because we need to show service status **on Configuration Service Modal** based on that.